stable/mail2most/values.yaml

```diff
@@ -0,0 +1,18 @@
+deployment:
+  replicas: 1
+  image:
+    repository: virtomize/mail2most
+    tag: latest
+    pullPolicy: Always
+
+resources: {}
+#  limits:
+#    cpu: 200m
+#    memory: 256Mi
+#  requests:
+#    cpu: 100m
+#    memory: 128Mi
+
+persistence:
+  enabled: true
```

Comment:
please add the other options for the persistence as well see this chart for an example: https://github.com/helm/charts/blob/master/stable/chartmuseum/values.yaml#L179 or any other chart in the stable folder

Reply:
@cpanato can you take a look if this is correct ?